internal/endpoints/gitlab.go

```diff
@@ -0,0 +1,206 @@
+package endpoints
+
+import (
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+
+	"fmt"
+
+	"github.com/redhatinsights/platform-changelog-go/internal/config"
+	"github.com/redhatinsights/platform-changelog-go/internal/db"
+	l "github.com/redhatinsights/platform-changelog-go/internal/logging"
+	"github.com/redhatinsights/platform-changelog-go/internal/metrics"
+	m "github.com/redhatinsights/platform-changelog-go/internal/models"
+	"github.com/xanzy/go-gitlab"
+)
+
+func GetURL(p *gitlab.PushEvent) string {
```

Comment:
Since these functions are just used in this package, you can use lowercase here. That way they wont' be exported for use in other packages. Go uses a capital first letter as an indicator that a function can be access from somewhere else